lighthouse-core/gather/connections/cri.js

```diff
@@ -108,16 +108,17 @@ class CriConnection extends Connection {
         });
       });
 
-      request.setTimeout(CONNECT_TIMEOUT, () => {
-        request.abort();
+      let ignoreECONNRESET = false;
```

Comment:
can you add a comment explaining this

Reply:
as in move up the comment that's where we set it or a new one duplicating? :)

Reply:
duplicate-ish?

Reply:
refactored so it's not necessary instead

Reply:
love it